services/api/service.go

```diff
@@ -912,6 +925,56 @@ func (api *RelayAPI) updatedExpectedRandao(slot uint64) {
 	}
 }
 
+// updatedExpectedWithdrawals updates the withdrawals field we expect from builder block submissions
+func (api *RelayAPI) updatedExpectedWithdrawals(slot uint64) {
+	api.log.Infof("updating withdrawals root for %d ...", slot)
```

Comment:
slot is logged with very log call here, might make sense to create a separate logger instance here that has this field baked in

Reply:
cool!